include/capstone/alpha.h

```diff
@@ -0,0 +1,280 @@
+#ifndef CAPSTONE_ALPHA_H
+#define CAPSTONE_ALPHA_H
+
+/* Capstone Disassembly Engine */
+/* By Nguyen Anh Quynh <aquynh@gmail.com>, 2014 */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if !defined(_MSC_VER) || !defined(_KERNEL_MODE)
+#include <stdint.h>
+#endif
+
+#include "../../cs_operand.h"
+#include "platform.h"
+
+#ifdef _MSC_VER
+#pragma warning(disable : 4201)
+#endif
+
+//> Operand type for instruction's operands
+typedef enum alpha_op_type {
+	ALPHA_OP_INVALID = CS_OP_INVALID, ///< CS_OP_INVALID (Uninitialized).
+	ALPHA_OP_REG = CS_OP_REG,	    ///< CS_OP_REG (Register operand).
+	ALPHA_OP_IMM = CS_OP_IMM,	    ///< CS_OP_IMM (Immediate operand).
+} alpha_op_type;
+
+// Instruction operand
+typedef struct cs_alpha_op {
+	alpha_op_type type; // operand type
+	union {
+		unsigned int reg; // register value for REG operand
+		int32_t imm; // immediate value for IMM operand
+	};
+} cs_alpha_op;
+
+// Instruction structure
+typedef struct cs_alpha {
+	// Number of operands of this instruction,
+	// or 0 when instruction has no operand.
+	uint8_t op_count;
+	cs_alpha_op operands[8]; // operands for this instruction.
```

Comment:
Move the `8` to its own `#define MAX_ALPHA_OPS 8`. Will be useful in the future.